elide-graphql/src/main/java/com/yahoo/elide/graphql/GraphQLConversionUtils.java

```diff
@@ -44,6 +46,11 @@
     protected NonEntityDictionary nonEntityDictionary = new NonEntityDictionary();
     protected EntityDictionary entityDictionary;
 
+    private final Map<Class, GraphQLObjectType> outputConversions = new HashMap<>();
```

Comment:
Maybe we should move these maps into the conversion ledger?

Reply:
that may be appropriate. I'll have to look it over again; I thought we actually needed to look at some of this stuff outside of the conversion ledger. In any case, refactoring to combine the two makes sense.